openquake/commonlib/nrml_examples/exposure-portfolio.xml

```diff
@@ -15,7 +15,7 @@
       <costTypes>
           <costType name="business_interruption" type="per_area" unit="EUR"/>
           <costType name="contents" type="per_area" unit="USD"/>
-          <costType name="non_structural" type="aggregated" unit="YEN" />
+          <costType name="nonstructural" type="aggregated" unit="YEN" />
```

Comment:
No, while manually testing the validator I discovered that this file in the examples directory was invalid. The file is not used by any test.